bootcd/create-image.sh

```diff
@@ -59,6 +59,23 @@ ns=`grep nameserver /etc/resolv.conf`
 nameserver 8.8.4.4'
 perl -pe "s/%%LocalNameservers%%/$ns/" genesis.ks.template > "$tmpdir/genesis.ks"
 
+# insert Extra Post Content into the %POST section of the ks file
+# used to build the image from file paths like /conf/*.content
+# which is bind mounted via: docker run -v /Some/Dir:/conf:ro ....
+if [[ $(find /conf -name \*.content 2>/dev/null | wc -l) > 0 ]]; then
+   echo '### inserting Extra Post Content'
+   sed -i -e "
```

Comment:
I tried running this `sed` blob but I don't think it produce the intended result. More testing is needed.